qa/rpc-tests/test_framework/util.py

```diff
@@ -130,11 +131,33 @@ def initialize_datadir(dirname, n):
         f.write("listenonion=0\n")
     return datadir
 
+def rpc_url(i, rpchost=None):
+    return "http://rt:rt@%s:%d" % (rpchost or '127.0.0.1', rpc_port(i))
```

Comment:
We should probably re-add that test to one of the standard sequence, otherwise it will just bitrot